pandas-stubs/core/generic.pyi

```diff
@@ -89,7 +90,7 @@ class NDFrame(PandasObject, indexing.IndexingMixin):
     def bool(self) -> _bool: ...
     def __abs__(self) -> NDFrame: ...
     def __round__(self, decimals: int = ...) -> NDFrame: ...
-    def __hash__(self): ...
+    __hash__: ClassVar[None]  # type: ignore[assignment]
```

Comment:
I think we should move this one up to the beginning of the class too.  Sorry for not catching it earlier.

Reply:
done and green again :)